src/libraries/Pausable.sol

```diff
@@ -71,19 +121,44 @@ abstract contract Pausable is Initializable {
         _;
     }
 
-    function __Paused_init() internal onlyInitializing {
-        __Paused_init_unchained();
+    /*
+     * @dev Modifier to allow only the Pauser to access pausing functionality
+     */
+    modifier onlyPauser() {
```

Comment:
We're not using this anywhere? Or do you want to keep this around for integrators to use?

Reply:
yea just keeping it for integrators